Data/models/DataController.swift

```diff
@@ -235,49 +237,24 @@ public class DataController {
         return DataController.sharedInMemory.container.viewContext
     }
     
-    static func save(context: NSManagedObjectContext?) {
-        guard let context = context else {
-            log.warning("No context on save")
-            return
-        }
-        
-        if context.concurrencyType == .mainQueueConcurrencyType {
-            log.warning("Writing to view context, this should be avoided.")
-        }
-        
-        context.perform {
-            if !context.hasChanges { return }
-            
-            do {
-                try context.save()
-            } catch {
-                assertionFailure("Error saving DB: \(error)")
-            }
-        }
-    }
-    
-    func addPersistentStore(for container: NSPersistentContainer, store: URL) {
-        let storeDescription = NSPersistentStoreDescription(url: store)
-        
-        // This makes the database file encrypted until device is unlocked.
-        let completeProtection = FileProtectionType.complete as NSObject
-        storeDescription.setOption(completeProtection, forKey: NSPersistentStoreFileProtectionKey)
-        
-        container.persistentStoreDescriptions = [storeDescription]
-    }
-    
     var storeURL: URL {
         let supportDirectory = Preferences.Database.DocumentToSupportDirectoryMigration.completed.value
         return supportDirectory ? supportStoreURL : oldDocumentStoreURL
     }
     
     private func configureContainer(_ container: NSPersistentContainer, store: URL) {
-        addPersistentStore(for: container, store: store)
-        
-        // Dev note: This completion handler might be misleading: the persistent store is loaded synchronously by default.
```

Comment:
Does this comment do not apply anymore ?

Reply:
not anymore, there's bunch of other synchronous completion handlers